includes/discovery/processors/stoneos.inc.php

```diff
@@ -0,0 +1,7 @@
+<?php
+if ($device['os'] === 'stoneos') {
+    $cpuUsage = snmp_get($device, 'HILLSTONE-SYSTEM-MIB::sysCPU.0', '-OvQU');
```

Comment:
Please pass the mib name on as an argument to the `snmp_get` call rather than using `HILLSTONE-SYSTEM-MIB::`